.travis.yml

```diff
@@ -23,22 +23,16 @@ addons:
       - build-essential
       - libsodium-dev
       - librocksdb5.17
-      - libsnappy-dev
```

Comment:
Y? What is then Snappy_lib_dir?

Reply:
See above comment, system snappy is unusable on Linux now. Updated wiki page as well.

Reply:
**What is then Snappy_lib_dir?**

Reply:
It is used only on Mac now, where the snappy is installed via homebrew. This is the list of Linux packages only

Reply:
Good, sorry, I missed it remains in macos-only section only, thanks.